index.js

```diff
@@ -62,6 +62,12 @@ const purchase = (productId, originalPurchaseToken) => {
   }
 };
 
+const restore = () => {
+  if (Platform.OS === "ios") {
```

Comment:
I think Android's in-app billing restoration is possible using methods like `queryPurchases`.
If you have time, would you like to implement it? Or should we save it for the next person?

Reply:
I don't know Android Development. That's why i didn't added

Reply:
Also some other problems available in this package.

Reply:
I Will try to fix it in free time.